src/Features/Core/Portable/QuickInfo/QuickInfoService.cs

```diff
@@ -16,13 +18,13 @@ public abstract class QuickInfoService : ILanguageService
         /// <summary>
         /// Gets the appropriate <see cref="QuickInfoService"/> for the specified document.
         /// </summary>
-        public static QuickInfoService GetService(Document document)
+        public static QuickInfoService? GetService(Document document)
```

Comment:
```suggestion
        public static QuickInfoService? GetService(Document? document)
```

Reply:
Good catch, and ugh that's a silly API shape.